INSTALL-LINUX.md

```diff
@@ -39,26 +39,15 @@ PyBaMM can be installed via pip:
 pip install pybamm
 ```
 
-
 PyBaMM has the following python libraries as dependencies: `numpy`, `scipy`, `pandas`,
-`matplotlib`. These will be installed automatically when you install PyBaMM using `pip`,
-following the instructions below. First, make sure you have activated your virtual
-environment as above, and that you have the latest version of pip installed:
-
-Then navigate to the path where you downloaded PyBaMM to (you will already be in the
-correct location if you followed the instructions above), and install both PyBaMM and
-its dependencies by typing:
+`matplotlib`. These will be installed automatically when you install PyBaMM using `pip`.
```

Comment:
This is hard to keep up to date with actual requirements and may be confusing, replace with e.g. "PyBaMM's optional dependencies (such as `numpy`, `scipy`, etc) will be installed automatically when you install PyBaMM using `pip`"